docs/content/build.md

```diff
@@ -1,46 +1,54 @@
 **Warning:** The [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
 
------
-## Building your workers
+---
 
-Before running a [deployment of your game](deploy.md), you need to:
-1. Build the bridge and launch configuration of your [workers](workers.md)
-2. Prepare the build configuration of your workers
-3. Build your workers
+## How to build your game
 
-To build your workers, you don't need to know what the bridge and launch configurations are, but if you do want to know more about them, see the SpatialOS documentation on [bridge configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#bridge-configuration) and [launch configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration#worker-launch-configuration).
+Before running a [deployment of your game](deploy.md) locally or in the cloud, you need to:
+
+1. Build your workers.
+2. Build the bridge configuration and launch configuration of your [workers](workers.md).
+3. Prepare the build configuration of your workers.
+
+### Building your workers
+
+To build your workers:
+
+* With your project open in the Unity Editor, select **SpatialOS** > **Build `WorkerName` for local** or **Build `WorkerName` for cloud** where `WorkerName` is the name of any worker you want to build.
+
+This starts the build process for the worker you selected and deployment type (cloud or local).
+
+> **It’s finished building when:** there is a message in the Unity Editor console.
+
+You can now find the built-out workers in `build/assembly/worker` in the root of the SpatialOS GDK for Unity project.
 
 ### Building the bridge and launch configuration of your workers
-To build the bridge and launch configurations for each of your server-workers and client-workers, you can either:
 
-* From the Unity Editor menu, select **SpatialOS** > **Build worker configs**.
+The [bridge](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration#worker-bridge-configuration) and [launch](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration#worker-launch-configuration) confgurations are pre-configued for you. You only need to build them. You do this for each of your [server-workers](https://docs.improbable.io/reference/latest/shared/glossary#server-worker) and [client-workers](https://docs.improbable.io/reference/latest/shared/glossary#client-worker). To do this you can either:
+
```

Comment:
Typo for "confgurations"  and "pre-configued" in this para.

Good effort from us (fist bump) for finding text for bit of a tricky thing to express (for the reasons we discussed). 

Looking at it, I would prefer it if we did say something about not needing to find out about the bridge and launcher because we want to try to keep using the GDK as simple as possible. I would suggest something along the lines of:

"The [bridge](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration#worker-bridge-configuration) and [launch](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration#worker-launch-configuration) configurations are pre-configured for you. You don't need to learn about them, you only need to build them. To do this..."

Reply:
"find out about them"?